pkg/cli/image/extract/extract.go

```diff
@@ -166,7 +171,7 @@ func NewExtract(streams genericclioptions.IOStreams) *cobra.Command {
 	flag.BoolVarP(&o.PreservePermissions, "preserve-ownership", "p", o.PreservePermissions, "Preserve the permissions of extracted files.")
 	flag.BoolVar(&o.OnlyFiles, "only-files", o.OnlyFiles, "Only extract regular files and directories from the image.")
 	flag.BoolVar(&o.AllLayers, "all-layers", o.AllLayers, "For dry-run mode, process from lowest to highest layer and don't omit duplicate files.")
-	flag.StringVar(&o.FileDir, "dir", o.FileDir, "The directory on disk that file:// images will be copied under.")
```

Comment:
Since there might be people using it, we can't just remove it, but rather deprecate this and provide information that users should use `--to-dir` instead.

Reply:
and since it does currently work as intended, we should probably _not_ deprecate but add the `--to-dir` and offer examples for both.